cPP_DSC.adoc

```diff
@@ -1622,10 +1622,7 @@ _If [.underline]#factory reset the TOE wiping out all non-permanent SDOs, as des
 
 FIA_SOS.2 TSF Generation of Secrets
 
-FIA_SOS.2.1:: The TSF shall provide a mechanism to generate *authorization data* that meet [_the following quality metrics:_
-+
-* _For each authentication attempt, the probability shall be less than one in 1,000,000 that a random attempt will be successful_
-* _For multiple attempts to authenticate during a one-minute period, the probability shall be less than one in 100,000 that a series of random attempts will be successful_].
+FIA_SOS.2.1:: The TSF shall provide a mechanism to generate *authorization data* that meet [_the metric where for each authentication attempt, the probability shall be less than one in 1,000,000 that a random attempt will be successful_].
```

Comment:
Looking at CC:2022, FIA_SOS.2 is really about generation of secrets. In this case, we're generating authorization data, so the secret is of course used for authentication. However, I don't think we should necessarily involve "authentication attempts" here, since I think it exceeds the scope of the SFR ever so slightly.

How about: "that meet _[the following quality metric: the probability of randomly guessing the value is less than one in 1,000,000]_"?

Of course, a random guess in this context is the same as an authentication attempt. But this wording decouples the process of authentication from the quality of the authorization data.

Reply:
I don't specifically have an issue with this, I was just trying to minimize the change from what was previously specified, hence the change to only remove the throttling part of the SFR.

Reply:
I'll open a separate PR after this one is merged